### ISSUE TITLE
Investigate migration to Swift 6

### ISSUE THREAD

**Issue by sosnovsky** [FlowCrypt/flowcrypt-ios#2625]
Xcode 16 includes Swift 6, which has some changes requiring migration of existing codebase - https://www.swift.org/migration/documentation/migrationguide/

Let's check if switching to Swift 6 will require too many changes for our existing code, or can be done quite quickly.

**Comment by ioanmo226** [comment]
I have been checking swift 6 migration and it doesn't seem like it would be done easily.
Need to migrate step by step and requires some time.

**Comment by ioanmo226** [comment]
Checking more now

**Comment by sosnovsky** [comment]
Yeah, it seems quite complicated, so no need to migrate for now, I removed `actionable` label and moved it to the next milestone.

**Comment by ioanmo226** [comment]
All assigned tasks assigned to me(except Swift6 upgrade is now finished)
Gonna work on browser extension but I can see that no tasks are assigned to me.
